project-reports/2023/2023-Q2-Hyperledger-Iroha.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: default
+title: 2023 Q2 Hyperledger Iroha
+parent: 2023
+grand_parent: Project Updates
+---
+
+# Project Health
+
+Currently, active maintenance of Iroha 1 has stopped and there are no plans for major releases. However, discussions to release the current RC continue because it will require more than one team switching from everyday tasks to focus on this.
+
+All Iroha 2 developers are now fully onboarded and at maximum productivity.
+
+Since development of the Ursa project has been discontinued, the Iroha team is rewriting its essential parts to integrate in the Iroha codebase. In addition, thanks to Arun S M, the Iroha 2 BCE team will start collaborating with the Hyperledger Blockchain Explorer team.
+
+# Questions/Issues for the TOC
+
+None.
+
+# Releases
+
+Past quarter releases:
+
+* v2.0.0-pre-rc.13 (internal)
+* v2.0.0-pre-rc.14 (internal)
+* v2.0.0-pre-rc.15 (internal)
+
+# Overall Activity in the Past Quarter
+
+According to the [Iroha Activity Dashboard](https://insights.lfx.linuxfoundation.org/projects/hyperledger%2Firoha/dashboard;subTab=technical?time=%7B%22from%22:%222023-01-01T07:00:00.000Z%22,%22type%22:%22absolute%22,%22to%22:%222023-03-31T07:00:00.000Z%22%7D) for the first quarter of 2023 (January-March), the Iroha repo has 129 PRs submitted by 28 contributors.
+
+## Iroha v1
+
+We are restoring the old documentation for the Iroha v1 project using a different hosting solution, as the previous solution was terminated. This is going to be used temporarily for support of the deprecated version of Iroha v1 documentation. 
```

Comment:
Q: Are these markdown files? Seems like a problem we could solve through right measures in place across the Hyperledger Foundation.

Reply:
Thanks, but sadly, Iroha 1 was using ReadTheDocs.
Until the I1 release is happening, I have moved the newer fork of the docs to https://iroha-1-fork.readthedocs.io/en/latest/, so I have more control over it.

What measures do you propose, though?

Reply:
I do not have a recommendation yet. But this looks like a problem statement for the documentation task force. My thoughts were in the lines of foundation providing the space for hosting the documentation (at least for the graduated projects).

Reply:
You can use the https://github.com/hyperledger-labs/documentation-template and host your documentation on github.io (examples: https://labs.hyperledger.org/documentation-template/latest and https://aca-py.org/main/)

Reply:
Thanks. Iroha 1 is not supported anymore. At the same time, I believe it'll be useful to contact @baziorek and discuss if I can help move the documentation to GitHub sites.